Zend/zend_virtual_cwd.h

```diff
@@ -68,10 +68,12 @@ typedef unsigned short mode_t;
 /* COPY_WHEN_ABSOLUTE is 2 under Win32 because by chance both regular absolute paths
    in the file system and UNC paths need copying of two characters */
 #define COPY_WHEN_ABSOLUTE(path) 2
```

Comment:
This seems no longer true, and likely would need to be adapted.

Reply:
The only usage in PHP is on line `Zend/zend_virtual_cwd.c(1415):`.

For linux it is zero and in the code `Zend/zend_virtual_cwd.c` it is incremented then, so `2` seems like ok to me.

But the comment seems very wrong, as UNC path like `//` does not make much more sense (UNC path without host).

In this commit https://github.com/php/php-src/pull/5001/commits/e3cd4c96dfde90ea00c9f64c64b649d8da85cfa4 I tried to set it to 3.

As this definition is hard to understand, is it ok to remove it?

Reply:
@cmb69 The only usage in PHP is on line `Zend/zend_virtual_cwd.c(1415):`.

For linux it is zero and in the code `Zend/zend_virtual_cwd.c` it is incremented then, so `2` seems like ok to me.

But the comment seems very wrong, as UNC path like `//` does not make much more sense (UNC path without host).

In this commit https://github.com/php/php-src/pull/5001/commits/e3cd4c96dfde90ea00c9f64c64b649d8da85cfa4 I tried to set it to 3.

As this definition is hard to understand, is it ok to remove it?

Reply:
Not sure, but that *might* be used by (external) extensions.

Reply:
@cmb69 I just finished my experiments and the current usage of `COPY_WHEN_ABSOLUTE()` is fully ok.